client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -231,23 +274,49 @@ class RemoveDomainDialog extends Component {
 	render() {
 		const { purchase, translate, chatButton } = this.props;
 		const productName = getName( purchase );
-		const buttons = [
-			{
-				action: 'cancel',
-				disabled: this.props.isRemoving,
-				isPrimary: true,
-				label: translate( 'Never Mind' ),
-			},
-			{
-				action: 'remove',
-				additionalClassNames: [
-					this.props.isRemoving || this.state.isCheckingEmail ? 'is-busy' : '',
-					'is-scary',
-				],
-				label: translate( 'Delete this Domain' ),
-				onClick: this.nextStep,
-			},
-		];
+		const buttons = ! config.isEnabled( 'domains/settings-page-redesign' )
+			? [
+					{
+						action: 'cancel',
+						disabled: this.props.isRemoving,
+						isPrimary: true,
+						label: translate( 'Never Mind' ),
+					},
+					{
+						action: 'remove',
+						additionalClassNames: [
+							this.props.isRemoving || this.state.isCheckingEmail ? 'is-busy' : '',
+							'is-scary',
+						],
+						label: translate( 'Delete this Domain' ),
+						onClick: this.nextStep,
+					},
+			  ]
+			: [
+					{
+						action: 'cancel',
+						disabled: this.props.isRemoving,
+						label: this.state.step === 3 ? translate( 'Nevermind' ) : translate( 'Cancel' ),
+					},
+					{
+						action: 'remove',
+						additionalClassNames: [
+							this.props.isRemoving || this.state.isCheckingEmail ? 'is-busy' : '',
+							'dialog__button--domains-remove',
+						],
+						isPrimary: true,
+						label:
+							this.state.step === 3 ? (
+								<>
+									<Icon icon={ trash } size={ 18 } />
+									{ translate( 'Delete this domain' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Delete this Domain' )` *ES Score: 8*